gradle/configs/git/hooks/pre-push.sh

```
@@ -81,13 +82,18 @@ if [ "$HAS_WEB" = "1" ]; then
-        if [ -n "$DOC_FILES" ]; then
-            lychee --config "lychee.toml" --offline --no-progress $DOC_FILES || true
-        fi
+if [ "$HAS_DOCS" = "1" ] && command -v lychee >/dev/null 2>&1; then
+    echo "📖 Running doc link check (changed docs detected)..."
+    DOC_FILES=$(echo "$CHANGED_FILES" | grep -E '\.(md|mdx)$' || true)
+    if [ -n "$DOC_FILES" ]; then
+        old_ifs=$IFS
+        IFS='
+'
+        set -f
+        set -- $DOC_FILES
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)